content/ADR-204-comms-architecture.md

```diff
@@ -0,0 +1,620 @@
+---
+layout: adr
+adr: 204
+title: Comms Architecture for new client
```

Comment:
I strongly suggest not putting the `2.0` nor `new client` here since this protocol is used in clients. Let's continue versioning the comms, the next version would be v5